semantic-conventions/src/opentelemetry/semconv/model/semantic_attribute.py

```diff
@@ -314,6 +315,7 @@ def check_stability(stability_value, position):
             "deprecated": StabilityLevel.DEPRECATED,
             "experimental": StabilityLevel.EXPERIMENTAL,
             "stable": StabilityLevel.STABLE,
+            "frozen": StabilityLevel.FROZEN,
```

Comment:
"Frozen" is not a stability level AFAIK but orthogonal to it. https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/document-status.md#feature-freeze

Reply:
trying to understand the concern: is it a different term or you'd rather see it as a separate flag?

1. If it's a term,  we chatted with @reyang and @trask  about `frozen` vs `feature-freeze` and considered (in the long term) changing the `feature-freeze` to `frozen` to apply better to things like attributes and be an adjective following `stable` or `deprecated` pattern.  Go use [`Frozen`](https://zchee.github.io/golang-wiki/Frozen/) convention for packages to describe that no new features are accepted.
2. If you'd rather see it as a separate flag: `feature-freeze` is still a valid document status used interchangeably with `Stable` or `Experimental`. Having two attribute parameters would be confusing, as I can write something meaningless as
  ```yaml
  stability: stable | deprecated
  feature-freeze: false
  ```

Reply:
2 is what I mean. As you can see in the linked document, a spec document has a lifecylce status and it can be feature-frozen or not. So indeed:

```yaml
stability: stable | deprecated | experimental(default)
feature-freeze: true | false(default)
```

At least I would suggest adopting the same system for whole spec documents and semantic convention elements, otherwise it could get confusing.

I have no strong opinion on whether that is ideal or if "frozen" should actually be a lifecylce status (optional step between experimental and stable?), but then I think it should be changed in the specs document-status spec first (which itself doesn't state a document status which would make it experimental, whether intentional or not)

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/document-status.md

Reply:
https://github.com/open-telemetry/opentelemetry-specification/pull/3456